### PR TITLE
UI update

### PR DESCRIPTION
**Pull request by aaronm759**
Visual update to the orders page. I also rearranged the css into a single folder.

### CODE REVIEW

**Comment by thadk** [comment]
Thanks Aaron, please fix this warning which is an error that was reported by the deploy preview tool above:
```2:18:14 PM: src/components/OrdersList.js
2:18:14 PM:   Line 9:10:  'currentIndex' is assigned a value but never used  no-unused-vars```

**Comment by aaronm759** [comment]
ok. me and rachel are trying to get our code to not conflict we each other, once that is done i will push it up

**Comment by thadk** [comment]
Thanks it's really helpful to have the deploy preview.

### Please update these things 
- [ ] increase margin, decrease drop shadow settings so it doesn’t run into next item unless that is meaningful somehow.
Consider only showing drop shadow on the item which is selected.
- [ ] Show the status code (`HOSS_blah`) to left or right without removing the description completely
- [ ] Add padding
- [ ] Please use bold/emphasis on titles of each line here (I think it is this way now)
- [ ] Format dates more like before (optionally you can use https://github.com/d3/d3-time-format )
- [ ] Check to make sure that someone's sorting functionality is not lost if they added that.
- [ ] Try to make buttons scan/edit etc horizontal in case we want to add more in the future.

image

File: src/components/TableHeader.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
+import { useState } from "react";
 import { SortArrows } from "./Sort/SortArrows";
+import styles from "../style/sort.module.css"
 
 export const TableHeader = (props) => {
+
+  const [sortControl, setSortControl] = useState('none')
```
Comment (thadk) [inline src/components/TableHeader.js:7]: Thanks for bringing these back in a way that made sense for the new design!

**Review by thadk — APPROVED** [review]
Nice job on the extra tweaks to bring it all together!

**Pull request merged**